modules/gateway/gateway_test.go

```diff
@@ -0,0 +1,76 @@
+package gateway
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/consensus"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+var port = 9001
```

Comment:
I very badly want to create some higher level testing package that'll allow us to get rid of this forever.